### PR TITLE
New !ytmix and !ytmusicmix commands to play YouTube mixes

### PR DESCRIPTION
**Pull request by javdc**
New commands to play YouTube auto-generated mix playlists for videos and songs. This can be done easily thanks to YouTube always using the the same url format for the mixes:

`https://www.youtube.com/watch?v={video_id}&list=RD{video_id}`

If a search query is used it'll be searched and the mix of the first result will be played.

This will obviously only work with single YouTube videos or songs, not with playlists.

### CODE REVIEW

**Review by candradesm — CHANGES_REQUESTED** [review]
This will be stalled until #42 is merged and this commands will be merged into the same !play.

File: main.py [hunk 1]
```diff
@@ -289,13 +291,51 @@ async def youtubemusic(ctx: Context, *args: str):
...
+
+
+async def play_youtube_mix(ctx: Context, yt_dlp_info: Any):
+    if yt_dlp_info is not None and yt_dlp_info.get('id') is not None:
+        title = yt_dlp_info.get('title')
+        video_id = yt_dlp_info.get('id')
+        await ctx.send(f":twisted_rightwards_arrows: Añadiendo el mix de `{title}`...")
+        await play(ctx, f"https://www.youtube.com/watch?v={video_id}&list=RD{video_id}")
+    else:
+        await ctx.send(":no_entry_sign: No se ha encontrado ningún contenido.")
```
Comment (Wikijito7) [inline main.py:338]: This block should be on main.py, move it to youtube.py. Also, don't use context outside of the command's function.

**Review by Wikijito7 — COMMENTED** [review]
This will be stalled until https://github.com/Wikijito7/ECIBot/pull/42 is merged and this commands will be merged into the same !play.

File: main.py (hunk 1 above)
Comment (javdc) [reply main.py:338]: This would need a big refactor in play command, better to do this later in another PR

**Pull request merged**